### DISCUSSION TITLE
WhatsApp Business API to BLOCK numbers

### DISCUSSION THREAD

**Discussion by xAli-1** [chatwoot/chatwoot#6818] [Q&A]
I got spam messages on my WhatsApp Business number from international numbers.

my question is, can i use [chatwoot](https://www.chatwoot.com/docs/product/channels/whatsapp/whatsapp-cloud) to auto block any phone number that is international number and not in my contact list?

**Comment by iammattmartin** [comment]
Not at this time. It has been raised before but like this and email addresses, there is no way to block anyone. Very annoying; mute is not a workable alternative.
